jerry-core/ecma/operations/ecma-exceptions.h

```diff
@@ -28,6 +28,12 @@
  * @{
  */
 
+#ifndef JERRY_NDEBUG
```

Comment:
This patch does not take effect on conformance. The error messages are always implementation defined, so they could be empty. There is a `message` property for the error object, but it contains empty string. I think this not a problem when you flash on mcu. In development you could use debug build for checking whats wrong. Further more we currently use really less error messages. After this PR we are able to provide (more descriptive) messages for errors without any negative effect on binary size in release builds.